apps/web/pages/booking/[uid].tsx

```diff
@@ -1013,15 +1013,6 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     customInputs: customInputSchema.array().parse(eventTypeRaw.customInputs),
   };
 
-  if (eventType.successRedirectUrl && isSuccessBookingPage && !cancel) {
```

Comment:
Don't redirect here as server isn't aware of whether the page is embedded or not.